schema/fs.go

```diff
@@ -379,735 +452,9107 @@ hR+PkX5zgcX30ucJvpgCXqksmwP73wn8U2qZo81xwnEy+6yyZ6Vexsl79az+DgAA//850zAaggUAAA==
 
 	"/fs.go": {
```

Comment:
Not really, since you're supposed to be [ignoring `*.go`][1].  Your copy of `esc` just seems to ignore `-ignore=…` :p.  For example, here's [my most-recently-landed `fs.go` rebuild][2], and it only contains `*.json`.

[1]: https://github.com/vbatts/oci-image-spec/blob/ab461b048bd1c8b6077d8e96936f706a518233c2/schema/gen.go#L21
[2]: https://github.com/opencontainers/image-spec/blob/10dc107a303d3118a1c452ec59192bd20510559d/schema/fs.go